identity/app/views/profile/privacyForm.scala.html

```diff
@@ -110,9 +120,10 @@ <h2>Thank you</h2>
     <fieldset class="fieldset fieldset--manage-account-noborder">
         <div class="fieldset__heading">
             <h2 class="form__heading">How else can we get in touch with you?</h2>
+            <p class="form__note">Besides email, are there any other ways you would like us to get in touch with you?</p>
```

Comment:
Is this redundant with the first line? maybe can be trimmed down but this one is down to opinion, don't really mind.